### ISSUE TITLE
Version 4.5.0 not found in Debian repository

### ISSUE THREAD

**Issue by SolarLiner** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3007]
**OS:** Ubuntu 17.10

**Issue Descriptions:** `apt` shows latest available version of GPMDP to be version 4.4.0; upgrading is only possible by manually installing the latest version from GitHub Releases, which kind of defeats the purpose of the Bintray repository.

**Comment by jostrander** [comment]
Agreed. Seems the windows store has suffered the same fate and is stuck on 4.4.0.

**Comment by MarshallOfSound** [comment]
Hm,I've definitely uploaded 4.5.0 to both places :thinking:

**Comment by jostrander** [comment]
So 4.5.0 is only available in the 32 bit repository for trusty. The debian repository has no mention of 4.5.0 and trusty 64 bit is also missing 4.5.0.

**Comment by jostrander** [comment]
I don't have access to bintray, so I can't be sure how to fix this issue. Assigning to @MarshallOfSound.

**Comment by juliencampion** [comment]
I'm using the Debian repository as well. There is still no update available today. Any update on the advancement of, well, updates, on the repository? Thanks in advance.

**Comment by Walkman100** [comment]
Also using the debian repository, latest version I get is `4.4.0-1`.

**Comment by calvin-sykes** [comment]
This still isn't fixed. It looks like the problem is that while you point users to download the 'debian' repository [here](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/blob/master/docs/Installation_On_Debian.md), you've added the newer packages to the old 'trusty' repository (compare [http://dl.bintray.com/marshallofsound/deb/dists/debian/main/binary-amd64/Packages](http://dl.bintray.com/marshallofsound/deb/dists/debian/main/binary-amd64/Packages) with [http://dl.bintray.com/marshallofsound/deb/dists/trusty/main/binary-amd64/Packages](http://dl.bintray.com/marshallofsound/deb/dists/trusty/main/binary-amd64/Packages).

So this should be relatively easy to fix, and it'd be great if it could be because at the moment the app is unusable when installed via apt as the 4.4.0 version has the "stuck at Loading Music Library Issue". I assume only the owner of the bintray account (@MarshallOfSound ?) is able to sort this though?
